### PR TITLE
[PLAY-1257] Ruby 3.3.0 Upgrade

### PR DESCRIPTION
**Pull request by jasperfurniss**
**What does this PR do?** A clear and concise description with your runway ticket url.
https://nitro.powerhrg.com/runway/backlog_items/PLAY-1257

✅  Upgrades Ruby in Playbook from 3.1.2 to 3.3.0
✅  Upgrades Ruby in Playbook Website from 3.1.2 to 3.3.0
✅  Upgrades Ruby in Examples from 3.1.2 to 3.3.0
✅  [Upgrade Instructions for Devs ](https://gist.github.com/jasperfurniss/03145176e9dd419ead7c3b9846bbb4dd)

Screenshot 2024-03-19 at 2 20 52 PM

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

#### Checklist:
- [X] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [X] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
~- [ ] **TESTS** I have added test coverage to my code.~

### CODE REVIEW

File: playbook-website/Gemfile.lock [hunk 1]
```diff
@@ -150,11 +151,8 @@ GEM
     net-smtp (0.3.3)
       net-protocol
     nio4r (2.5.9)
-    nokogiri (1.14.2-arm64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.14.2-x86_64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.14.2-x86_64-linux)
+    nokogiri (1.14.2)
```
Comment (markdoeswork) [inline playbook-website/Gemfile.lock:154]: did this need to be upgraded because of the ruby version? just curious

**Review by markdoeswork — APPROVED** [review]
💎

File: playbook-website/Gemfile.lock (hunk 1 above)
Comment (jasperfurniss) [reply playbook-website/Gemfile.lock:154]: @markdoeswork 

Great question. It definitely upgraded on it's own.
I just ran ./setup and let bundler upgrade as it saw fit. cc: @markdoeswork 

👍  We'll test it and see if there are any issues. 
In fact, would you mind sanity-checking my PR, by pulling down my branch, and running the commands:
1. `asdf plugin-update ruby`
2. `asdf install ruby 3.3.0`
3. `./setup` OR `bundle && yarn`
4. `yarn start-dev`
5.  Confirm the site loads. :)

Comment (markdoeswork) [reply playbook-website/Gemfile.lock:154]: thats for the deets

**Review by thestephenmarshall — APPROVED** [review]

**Pull request merged**